### ISSUE TITLE
Table displays no records after importing a SQLite file when a filter is set

### ISSUE THREAD

**Issue by johnmhoran** [nexB/aboutcode-manager#181]
On both Windows 10 and Ubuntu 16.04, I experienced the following behavior when I tested `v2.0.0-rc5.post8.1efb837`:

* After launching AboutCode Manager, I imported a `zlib-1.2.11` scan I ran several days ago and saved it as a `sqlite` file.  Navigation and performance looked good.

* With `zlib` open, I imported and saved as `sqlite` a `dropbear-2017.75` scan I also ran several days ago.

* With `dropbear` now open, when I navigated to the table view, the root node was selected, but no records were displayed in the table -- no records found.  Same behavior as I navigated through the tree.  I then navigated to the bar chart, selected `Copyright Authors`, selected one of the entries, and the table view displayed records for that entry, but still no records as I selected other nodes in the tree.

* I closed and relaunched AboutCode Manager.  This time, I did not import any ScanCode JSON files.  Instead, I selected the `zlib` `sqlite` file I'd just created, and the tree nodes displayed records in the table view as expected.  Same result for `dropbear`.

* I've attached a `.zip` containing the `zlib` and `dropbear` scans I was working with.

[zlib_and_dropbear_scans.zip](https://github.com/nexB/aboutcode-manager/files/1411624/zlib_and_dropbear_scans.zip)

**Comment by mjherzog** [comment]
This is not likely a solution to this problem, but it would be good to have an option to "Close SQLite File" from the File menu. This is a very common app option and would allow a user to switch between Files without closing ABC Manager.

**Comment by jdaguil** [comment]
Closing this for now. @johnmhoran and I could not reproduce this using `2.0.0-rc5.post8.1efb837`

**Comment by johnmhoran** [comment]
@jdaguil  -- After further testing yesterday, I was able to reproduce the behavior I reported when I first opened this issue but was unable to replicate when I tried to demonstrate it to you last week.  

On Windows 10 with `Release v2.0.0`:
* Create or open a `sqlite` file.
* Navigate to the bar chart and select a bar/attribute.
* The app takes the user to the tree/table view and displays the responsive record(s) using a column filter.
* Create or open a different `sqlite` file.
* The file opens at the tree/table view with the root node selected, and with the previous file's column filter still set.  Unless the filter identifies a record in the new file, the table view displays "No matching records found".
* Until the user notices that the filter is still set and clears it, selecting other nodes in the tree likely will display the same message.

Possible resolutions:
* Clear all column filters when a user chooses to open an existing `sqlite` file or create a new one by importing a `JSON` file.
* It might be handy to highlight active table-column filters (e.g., with the font or background color).
